### PR TITLE
fix(Tools/vmaps): Fix extractor crash when some .mpq files does not exist

### PR DESCRIPTION
**Pull request by IntelligentQuantum**
## Changes Proposed:
-  
-  

## Issues Addressed:

- Closes 

## SOURCE:

https://github.com/TrinityCore/TrinityCore/commit/3d7a86bf1e7596b040ebaada23d377368096da0f
## Tests Performed:

- 
- 

## How to Test the Changes:

1.
2.
3.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by acidmanifesto** [comment]
@IntelligentQuantum can you fetch the upstream and see if the codefactor issue is resolved? its literally the only thing preventing the merge button from showing.

**Review by acidmanifesto — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -65,6 +65,7 @@ nbproject/
 .idea
 cmake-build-debug/*
 cmake-build-debug-coverage/*
+cmake-build-debug-event-trace/*
```
Comment (acidmanifesto) [inline .gitignore:68]: what is the purpose of this one?

**Review by acidmanifesto — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (IntelligentQuantum) [reply .gitignore:68]: Related to Clion IDE

**Review by acidmanifesto — APPROVED** [review]

**Comment by acidmanifesto** [comment]
Tested with select mpq's being removed, no crash. Will get with other staffers in merging it. No new map extractions is required.

**Pull request merged**